table-api/src/main/java/io/deephaven/api/agg/Aggregation.java

```diff
@@ -655,6 +671,21 @@ static Sentinel Sentinel() {
         return Sentinel.of();
     }
 
+    /**
+     * Calls every single visit method of {@code visitor} with a {@code null} object.
+     *
+     * @param visitor the visitor
+     */
+    static void visitAll(Visitor visitor) {
```

Comment:
I think that this method and `AggSpec.visitAll` should move to test code. I can't think of a "production" scenario.